app/connectors/exchanges/IleQueryRequest.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors.exchanges
+
+import models.UcrBlock
+import play.api.libs.json.Json
+
+case class IleQueryRequest(eori: String, providerId: Option[String], ucrBlock: UcrBlock)
```

Comment:
All other Exchange classes has `providerId` as just `String`. Is there a particular reason that it is optional here?
And one more thing: for consistency, maybe it would be better to call this class `IleQueryExchange`?

Reply:
I used this to be aligned with the frontend. Good point. I changed to be just a String and changed name.